invenio/modules/authors/recordext/fields/author_base.cfg

```diff
@@ -65,7 +65,7 @@ name:
          'schema': {'last': {'type': 'string', 'empty': False},
                     'first': {'type': 'string', 'empty': False}}}}
     creator:
-        marc, "100__", {'last': util_split(value['a'], ', ', 0),
+        marc, "100._", {'last': util_split(value['a'], ', ', 0),
```

Comment:
To my understanding: `.` is wildcard for *any character*. `_` means instead *no character*.

The fact that you have `._` is because in MARC you have two indicators: in this case the first one can be anything while the second one need to be empty.